plugins/modules/netbox_power_outlet_template.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# © 2020 Nokia
+# Licensed under the GNU General Public License v3.0 only
+# SPDX-License-Identifier: GPL-3.0-only
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: netbox_power_outlet_template
+short_description: Create, update or delete power outlets within Netbox
+description:
+  - Creates, updates or removes power outlets from Netbox
+notes:
+  - Tags should be defined as a YAML list
+  - This should be ran with connection C(local) and hosts C(localhost)
+author:
+  - Tobias Groß (@toerb)
+requirements:
+  - pynetbox
+version_added: '0.2.2'
```

Comment:
If you can get the changes in by Sunday, can you change version to `0.2.3`, but if you can't get the changes in by Sunday, change it to `0.2.4`

Reply:
fixed to 0.2.3.